java/test/jmri/util/ValidatingInputPaneTest.java

```diff
@@ -0,0 +1,217 @@
+package jmri.util;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertFalse;
+import static junit.framework.TestCase.assertNotNull;
+import static junit.framework.TestCase.assertNull;
+import static junit.framework.TestCase.assertTrue;
```

Comment:
You should be using .org.junit.Assert.  We should see anything in JUnit4 tests from junit.framework.